### ISSUE TITLE
Model.evaluate_generator returns the loss of last batch data

### ISSUE THREAD

**Issue by loribeehive** [keras-team/keras#13780]
<em>Please make sure that this is a Bug or a Feature Request and provide all applicable information asked by the template.
If your issue is an **implementation question**, please ask your question on [StackOverflow](http://stackoverflow.com/questions/tagged/keras) or [on the Keras Slack channel](https://keras-slack-autojoin.herokuapp.com/) instead of opening a GitHub issue.</em>  

**System information**  
- Have I written custom code (as opposed to using example directory):  Yes
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04):  MacOS
- TensorFlow backend (yes / no):  yes
- TensorFlow version:  2.1.0
- Keras version:  2.3.1
- Python version:  3.7
- CUDA/cuDNN version:  
- GPU model and memory:  N/A

You can obtain the TensorFlow version with:  
python -c "import tensorflow as tf; print(tf.GIT_VERSION, tf.VERSION)"  
You can obtain the Keras version with:  
python -c 'import keras as k; print(k.__version__)'  

**Describe the current behavior**  
model.evaluate_generator returns the loss of only last batch of data
**Describe the expected behavior**  
should be returning the average/median over all the batches
**Code to reproduce the issue**  
Provide a reproducible test case that is the bare minimum necessary to generate the problem.  
score = model.evaluate_generator(eval_generator)
for batch in eval_batches:
      Loss.append(model_6.test_on_batch(batch,reset_metrics=False))

Code above gave:
score==Loss[-1]
**Other info / logs**  
Include any logs or source code that would be helpful to diagnose the problem. If including tracebacks, please include the full traceback. Large logs and files should be attached.  
The line of source code been referenced is only taking the last output of outs_per_batch:
 averages = [float(outs_per_batch[-1][0])]

**Comment by Saduf2019** [comment]
@loribeehive 
Moving this issue to closed status as there has been no activity, in case you still face the error please create a new issue.
